README.md

```diff
@@ -280,6 +280,8 @@ scaling:
   redis:
     host:
     password:
+    existingSecret:
+    existingSecretKey:
```

Comment:
I suggest changing that to `existingSecretPasswordKey`, as since Redis 6 you can authenticate with both username and password. I didn't check if this is supported by underlying ENV variables, but naming it like that makes the chart future-proof :)

Reply:
agree

Reply:
@sebastiansterk any feedback on your PR?

Reply:
@tomaszkiewicz @Vad1mo sounds good!

Reply:
Implemented.